### PR TITLE
Update cf versions

### PR DESCRIPTION
**Pull request by apinnick**
@cbudz I updated the CFME version numbers. Please review. Thanks

### CODE REVIEW

File: doc-Infrastructure_Migration_Solution_Guide/cfme/IMS_1.1/modules/proc_Installing_cf_4_7_0.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ Removing or changing a provider will cause errors in the infrastructure mappings
 
 ifdef::rhv[]
 [[Cloudforms_for_rhv]]
-. Install Red Hat CloudForms 4.7.3 (CFME 5.10.3) on the Manager machine. See link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html/installing_red_hat_cloudforms_on_red_hat_virtualization[Installing Red Hat CloudForms on Red Hat Virtualization].
+. Install Red Hat CloudForms 4.7.0 (with CFME 5.10.3) on the Manager machine. See link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html/installing_red_hat_cloudforms_on_red_hat_virtualization[Installing Red Hat CloudForms on Red Hat Virtualization].
```
Comment (cbudz) [inline doc-Infrastructure_Migration_Solution_Guide/cfme/IMS_1.1/modules/proc_Installing_cf_4_7_0.adoc]: Is there any particular reason that CF 4.7.0 is paired with 5.10.3 here?

File: doc-Infrastructure_Migration_Solution_Guide/cfme/IMS_1.1/modules/proc_Installing_cf_4_7_0.adoc [hunk 2]
```diff
@@ -26,7 +26,7 @@ CFME 5.10.4 does not support migration.
 endif::rhv[]
 ifdef::osp[]
 [[Cloudforms_for_osp]]
-. Install Red Hat CloudForms 4.7.3 (CFME 5.10.3). See link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html-single/installing_red_hat_cloudforms_on_red_hat_openstack_platform/[Installing Red Hat CloudForms on Red Hat OpenStack Platform].
+. Install Red Hat CloudForms 4.7.0 (with CFME 5.10.3 virtual appliance). See link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html-single/installing_red_hat_cloudforms_on_red_hat_openstack_platform/[Installing Red Hat CloudForms on Red Hat OpenStack Platform].
```
Comment (cbudz) [inline doc-Infrastructure_Migration_Solution_Guide/cfme/IMS_1.1/modules/proc_Installing_cf_4_7_0.adoc]: Same comment as above - 4.7.0/5.10.3?

File: doc-Infrastructure_Migration_Solution_Guide/cfme/IMS_1.1/modules/ref_Software_compatibility_matrix.adoc [hunk 3]
```diff
@@ -12,7 +12,7 @@
 ifdef::rhv[]
 |Red Hat Virtualization |4.2.8
 endif::rhv[]
-|CloudForms .<a|4.7.3, with CFME 5.10.3
+|CloudForms .<a|4.7.0, with the CFME 5.10.3 virtual appliance
```
Comment (cbudz) [inline doc-Infrastructure_Migration_Solution_Guide/cfme/IMS_1.1/modules/ref_Software_compatibility_matrix.adoc]: Same as above

File: doc-Infrastructure_Migration_Solution_Guide/cfme/IMS_1.1/modules/proc_Installing_cf_4_7_0.adoc (hunk 1 above)
Comment (apinnick) [reply doc-Infrastructure_Migration_Solution_Guide/cfme/IMS_1.1/modules/proc_Installing_cf_4_7_0.adoc]: 5.10.3 is the latest version that will support migration for CF 4.7.0, which can migrate RHV 4.2.
In 5.10.4, migration is completely broken, so we had to skip that one.
5.10.6 works with CF 4.7.6, which we can't use for IMS 1.1. RHV 4.2 did not support creating conversion hosts in CF UI, so we had to use Ansible for 1.1.

File: doc-Infrastructure_Migration_Solution_Guide/cfme/IMS_1.1/modules/proc_Installing_cf_4_7_0.adoc (hunk 2 above)
Comment (apinnick) [reply doc-Infrastructure_Migration_Solution_Guide/cfme/IMS_1.1/modules/proc_Installing_cf_4_7_0.adoc]: Probably less relevant for OSP. I'll ask.

File: doc-Infrastructure_Migration_Solution_Guide/cfme/IMS_1.1/modules/ref_Software_compatibility_matrix.adoc (hunk 3 above)
Comment (apinnick) [reply doc-Infrastructure_Migration_Solution_Guide/cfme/IMS_1.1/modules/ref_Software_compatibility_matrix.adoc]: Answer is same as above. Definitely necessary for RHV, but not sure it's required for OSP.

**Comment by cbudz** [comment]
Thanks for the feedback, merging this request

**Pull request merged**